### ISSUE TITLE
Help message for wasm-pack build is not updated

### ISSUE THREAD

**Issue by ibaryshnikov** [rustwasm/wasm-pack#636]
## 🐛 Bug description
In help message `target` still defaults to `browser`

More details:
`wasm-pack build --help` prints this:
```
-t, --target <target>        Sets the target environment. [possible values: browser, nodejs, web, no-modules]
                                 [default: browser]
```

In the [book](https://rustwasm.github.io/docs/wasm-pack/commands/build.html#target) the default is `bundler`

Actually, both are supported, but `--target browser` silently became deprecated
https://github.com/rustwasm/wasm-pack/blob/master/src/command/build.rs#L97

#### 🤔 Expected Behavior
`wasm-pack build --help` should print that `bundler` is the default target, and maybe include some information about deprecation of `browser` target

#### 🌍 Your environment
wasm-pack version: 0.8.1

I'd like to submit a pr if this update makes sense

**Comment by ashleygwilliams** [comment]
ah ha! i did Not Realize that that had changed in wasm-bindgen. thank you so much for filing an issue. i'd love a PR for this and it would be great to also update the documentation!

cc @alexcrichton is this assessment correct? is there anything else that we may have missed that updated in wasm-bindgen? @ibaryshnikov please go ahead an open up a PR and we can discuss anything else there!

**Comment by alexcrichton** [comment]
Oh I think this is just outdated documentation, AFAIK the default is "bundler" in both wasm-bindgen and wasm-pack

**Comment by ashleygwilliams** [comment]
closed by #640 by way of #678
